packages/stream-tile-handler/src/__tests__/tile-document.test.ts

```diff
@@ -587,6 +592,166 @@ describe('TileDocumentHandler', () => {
     delete state.metadata.unique
     expect(state).toMatchSnapshot()
   })
+
+  it('fails to apply commit if keys revoked', async () => {
```

Comment:
it's pretty hard to follow what exactly this test is doing.  Can we pull out some pieces into helper functions and add some more comments to make it clearer what we're doing at each phase of this test?  Same for other tests added

Reply:
yes can do